lib/wsd.js

```diff
@@ -91,18 +92,18 @@ class WSD {
    *   formats include: 'png', 'svg', or 'pdf'.  'pdf' requires a paid
    *   account.
    * @param {string} [apikey] API key for non-free usage.
+   * @param {string} [root] Root URL for the service.
```

Comment:
`[root='https://websequencediagrams.com']`

Reply:
I’ll be out for the next few hours, so if you can make that change, please do. Thanks!